### PR TITLE
Final lessons for Olomouc Autumn 2021

### PR DESCRIPTION
**Pull request by fivaldi**

### CODE REVIEW

**Comment by fivaldi** [comment]
@kobzol Please for review.

File: runs/2021/pyladies-olomouc-podzim/info.yml [hunk 1]
```diff
@@ -235,3 +235,38 @@ plan:
...
+
+  - title: Závěrečný projekt
+    slug: asteroids
+    date: 2022-01-20
+    materials:
+      - lesson: projects/asteroids
+        type: lesson
+      - lesson: projects/snake
+        type: lesson
+      - title: Množinový tahák
```
Comment (Kobzol) [inline runs/2021/pyladies-olomouc-podzim/info.yml:247]: Why is the set cheatsheet included here? :thinking:

Comment (fivaldi) [reply runs/2021/pyladies-olomouc-podzim/info.yml:247]: Asteroids project uses/references `set`

Comment (Kobzol) [reply runs/2021/pyladies-olomouc-podzim/info.yml:247]: Aha, okay :)

**Review by Kobzol — APPROVED** [review]

**Pull request merged**